fv3gfs-physics/fv3gfs/physics/stencils/physics.py

```diff
@@ -169,18 +167,15 @@ def __init__(
         stencil_factory: StencilFactory,
         grid_data: GridData,
         namelist,
-        comm: pace.util.CubedSphereCommunicator,
-        partitioner: TilePartitioner,
-        rank: int,
-        grid_info,
+        active_packages: List[str],
```

Comment:
Can you document what options are available for active_packages, either in a docstring or through a `List[Literal["microphysics", "another_package_name", ...]]` type hint?